docs/index.md

```diff
@@ -114,7 +114,29 @@ If the issuer has more than one certificate for the same public key (e.g. partic
 
 Issuers SHALL publish their public keys as JSON Web Key Sets (see [RFC7517](https://tools.ietf.org/html/rfc7517#section-5)), available at `<<iss value from JWS>>` + `/.well-known/jwks.json`, with [Cross-Origin Resource Sharing (CORS)](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin) enabled.
 
-The URL at `<<iss value from JWS>>` SHALL use the `https` scheme and SHALL NOT include a trailing `/`. For example, `https://smarthealth.cards/examples/issuer` is a valid `iss` value (`https://smarthealth.cards/examples/issuer/` is **not**).
+The URL at `<<iss value from JWS>>` SHALL use the `https` scheme, and SHALL use the following TLS version 1.2 or 1.3 configurations:
+* For TLS 1.2, use a ciphersuite conforming to the following algorithm choices and _minimum_ cryptographic strength:
+  * Key exchange: ECDHE using a 256-bit curve
+  * Authentication: ECDSA with a 256-bit curve, or RSA with a 2048-bit key
+  * Session cipher: AES with a 128-bit key
+  * Cipher mode: CBC or CGM
+  * Hash algorithm: SHA256
+  * The following recommended ciphersuites adhere to these requirements:
+    * TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
+    * TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
+    * TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+    * TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
+    * TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256
+    * TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA384
+    * TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256
+    * TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384
+* For TLS 1.3, use one of these ciphersuites:
+    * TLS_AES_256_GCM_SHA384
+    * TLS_AES_128_GCM_SHA256
+* X.509 certificate SHALL use a 256-bit ECDSA or 2048-bit RSA key, must be valid, and trusted by major web browsers.
```

Comment:
```suggestion
* X.509 certificate SHALL use at least a 256-bit ECDSA or at least a 2048-bit RSA key; SHALL be valid; and SHALL be trusted by major web browsers.
```

These are meant as minimums, right? (There's no outer bullet stating such at this point in the bullet list, so we might want to say so explicitly inline.)

Reply:
reword? The way the last phrase of the proposed change is worded suggests that this spec intends to dictate what 'major' web browsers shall or shall not do; also there is likely disagreement on what defines a 'major' browser, so that is not really enforceable/testable.

Reply:
Do you have a proposed wording? I was avoiding being explicit for X.509 requirements on purpose here, because this would require _a lot_ of text (what chain properties are valid? Rooting to what? Expiration? Revocation?) Browsers typically have a good handle of what a "valid" cert is; if you use a 'major' one to visit the completed issuer URL and get a green check mark, you are compliant; so could we phrase that in a vendor-neutral way?

Reply:
And RE: ECDSA vs EC, an auth X.509 cert can only have ECDSA alg, no? RSA is also a sig alg (confusing in this case, because it's name the same if used for encryption or signing).

Reply:
Superseded by ref to BCP 195